patches/minecraft/net/minecraft/world/level/block/FlowerPotBlock.java.patch

```diff
@@ -82,5 +82,12 @@
 +       }
 +       fullPots.put(flower, fullPot);
 +   }
++
++   public Map<net.minecraft.resources.ResourceLocation, java.util.function.Supplier<? extends Block>> getFullPotsView() {
++      if (fullPots == null) {
```

Comment:
fullPots is never null, it is either a new HashMap or a Collections.emptyMap()
This condition should probably be `if(this.emptyPot != null)` which implies that `this` is a full pot.
There is another question to be asked if this should even raise an exception instead of just returning the empty map.

Reply:
Will fix the condition, no idea how I overlooked that.

I decided to throw an exception to match the behaviour of `addPlant()` but I can remove the exception if that is deemed to be more appropriate.